server/routes/aboutPop/locale.json

```diff
@@ -3,9 +3,17 @@
     "page": {
       "title": "About - {{ applicationName }}"
     },
+    "header": {
+      "returnToOasysButton": "Return to OASys",
```

Comment:
I wonder if this should be moved to the common-locale?

Reply:
Since this branch started this got added to commonLocale.json as `header.returnToOasysButton`.

Reply:
changed